echopype/calibrate/calibrate_base.py

```diff
@@ -1,39 +1,21 @@
-import xarray as xr
-
-ENV_PARAMS = (
-    'temperature',
-    'salinity',
-    'pressure',
-    'sound_speed',
-    'sound_absorption'
-)
+ENV_PARAMS = ("temperature", "salinity", "pressure", "sound_speed", "sound_absorption")
 
 CAL_PARAMS = {
-    'EK': (
-        'sa_correction',
-        'gain_correction',
-        'equivalent_beam_angle'
-    ),
-    'AZFP': (
-        'EL',
-        'DS',
-        'TVR',
-        'VTX',
-        'equivalent_beam_angle',
-        'Sv_offset'
-    )
+    "EK": ("sa_correction", "gain_correction", "equivalent_beam_angle"),
+    "AZFP": ("EL", "DS", "TVR", "VTX", "equivalent_beam_angle", "Sv_offset"),
 }
 
 
 class CalibrateBase:
-    """Class to handle calibration for all sonar models.
-    """
+    """Class to handle calibration for all sonar models."""
 
     def __init__(self, echodata):
         self.echodata = echodata
         self.env_params = None  # env_params are set in child class
         self.cal_params = None  # cal_params are set in child class
-        self.range_meter = None  # range_meter is computed in compute_Sv/Sp in child class
+        self.range_meter = (
+            None  # range_meter is computed in compute_Sv/Sp in child class
+        )
```

Comment:
since this type of one argument line break happens a few times already, is it the default number of characters in one line that is causing this?

Reply:
https://black.readthedocs.io/en/stable/the_black_code_style/current_style.html